pos-print/src/main/java/com/clouway/pos/print/adapter/http/HttpModule.java

```diff
@@ -22,6 +22,7 @@ protected void configureSitebricks() {
         at("/_status").serve(StatusService.class);
         at("/v1/posprint").serve(PrintService.class);
         at("/v1/devices").serve(CashRegistersService.class);
+        at("/v1/report").serve(ReportService.class);
```

Comment:
@mgenov So, this will mean that we must have two separated services, one for operator reports and one for period reports?